.changeset/calm-oranges-speak.md

```diff
@@ -0,0 +1,6 @@
+---
+'@graphql-codegen/visitor-plugin-common': minor
+'@graphql-codegen/typescript-operations': minor
```

Comment:
Let me know if you'd prefer these to be `patch`.

Reply:
It is a new feature so `minor` seems adequate.